include/zephyr/drivers/gpio.h

```diff
@@ -875,8 +876,12 @@ static inline int z_impl_gpio_pin_interrupt_configure(const struct device *port,
 		(const struct gpio_driver_data *)port->data;
 	enum gpio_int_trig trig;
 	enum gpio_int_mode mode;
+	int ret;
+
+	sys_port_trace_gpio_pin_interrupt_configure_enter(port, pin, flags);
```

Comment:
why not use the macros? for example:

`SYS_PORT_TRACING_OBJ_FUNC_ENTER(gpio_pin_interrupt_configure, pin, flags)`
?

Reply:
Pushed a change to address this - PTAL